velox/exec/HashTable.cpp

```diff
@@ -359,6 +359,12 @@ void populateNormalizedKeys(HashLookup& lookup, int8_t sizeBits) {
 
 template <bool ignoreNullKeys>
 void HashTable<ignoreNullKeys>::groupProbe(HashLookup& lookup) {
+  ExceptionContextSetter exceptionContext(
```

Comment:
This is an unrelated change, right? Let's move it into a separate PR and add a test.